cli/ssh.go

```diff
@@ -1076,5 +1081,5 @@ func getUsageAppName(usageApp string) codersdk.UsageAppName {
 		return codersdk.UsageAppName(usageApp)
 	}
 
-	return ""
+	return codersdk.UsageAppNameSSH
```

Comment:
Letting the zero value mean 'disabled' feels more idiomatic to me, but 'disabled' is way more explicit and better from a user perspective.

Reply:
I considered having the default value on the flag be `ssh` and disable reporting if it's manually set to empty string but I wasn't sure if that was explicit enough behavior. I'm thinking in the extensions where we do `CODER_USAGE_APP_NAME="" coder ssh ...` I'm worried there may be cases where the empty string get's passed to the CLI as "unset" instead.